klog.go

```diff
@@ -756,25 +756,7 @@ func (l *loggingT) output(s severity, buf *buffer, file string, line int, alsoTo
 		if alsoToStderr || l.alsoToStderr || s >= l.stderrThreshold.get() {
 			os.Stderr.Write(data)
 		}
```

Comment:
Move this block into `writeLogData` too

Reply:
The writeLogData is also used in fatal error (see below "if s == fatalLog"), moving the "tostderr" block inside writeLogData will add unwanted log messages. I can try using a flag param, like "fatalerror".